src/Components/test/E2ETest/ServerRenderingTests/NoInteractivityTest.cs

```diff
@@ -127,36 +120,35 @@ private void Assert404ReExecuted() =>
     [Theory]
     [InlineData(true)]
     [InlineData(false)]
-    public void CanRenderNotFoundPageNoStreaming(bool useCustomNotFoundPage)
+    public void CanRenderNotFoundPage(bool streamingStarted)
```

Comment:
This one covers the old:
- `CanRenderNotFoundPageNoStreaming(useCustomNotFoundPage: true)`,
- `CanRenderNotFoundPageWithStreaming(useCustomNotFoundPage: true)` - we used to redirect to "not-found" route here, now the behavior is same as for no-streaming, so we can consolidate them.